i18n/fr.yaml

```diff
@@ -0,0 +1,32 @@
+- id: about
+  translation: "À propos"
+
+- id: home
+  translation: "Accueil"
+
+- id: recentPosts
+  translation: "Derniers billets"
+
+- id: readMore
+  translation: "Lire la suite →"
+
+- id: translations
+  translation: "Traductions"
+
+- id: share
+  translation: "Partager"
+
+- id: paginatorPrevious
+  translation: "Page précédente"
+
+- id: paginatorNext
+  translation: "Page suivante"
+
+- id: cookieMessage
+  translation: "Ce site web utilise des cookies"
+
+- id: cookieLink
+  translation: "En savoir plus."
+
+- id: cookieDismiss
+  translation: "J'ai compris !"
```

Comment:
LOL no problem. I love rules. I just noticed it immediately and thought I'd ask. 👍